templates/macros/menu.html

```diff
@@ -38,11 +38,43 @@
             <ul class="menu__inner">
             {%- for item in menu_items %}
                 <li {%- if current_item and current_item == item %} class="active" {%- endif %}>
-                    {%- if item.newtab -%}
-                        <a href="{{ item.url | replace(from="$BASE_URL", to=config.base_url) | safe }}" target="_blank" rel="noopener noreferrer">{{ item.name | safe }}</a>
+
+                    <!-- RSS -->
+                    {%- set is_rss = item.url == "$BASE_URL/$FEED_FILENAME" -%}
```

Comment:
We use

`{%- if feed is containing('rss') %}`

elsewhere now (see #71), would be good to keep it consistent.

Reply:
To do that, I'd need to do it inside the `{%- for feed in config.feed_filenames %}` loop, but this specific _if_ is guarding against a different thing: whether or not the current item refers to the RSS icon or not (we don't care about RSS or ATOM at this stage). If it is referring to RSS, then we can add the symbol and also add the relevant `<a>` element depending if it's ATOM or RSS. Does that make sense ?